.github/workflows/refresh_db.yml

```diff
@@ -2,8 +2,8 @@ name: DB Refresh (Objects API)
 
 on:
   schedule:
-#    - cron: '*/20 * * * *'  # every 20 minutes
     - cron: '0 5 * * *'  # every day 5am GMT (12-1am EST/EDT)
+#    - cron: '*/20 * * * *'  # every 20 minutes
```

Comment:
### GH action > backend directly
This experience reinforces my position that we should probably change the refresh from calling the backend directly back to using the GitHub action. As it applies to this current situation:

1. I don't have (reasonable) access to logs
I want to see not just the cronjobs, but also logs from when refresh button was used, and I can't see that in GitHub now.
2. The refresh button won't have fixes/updates until next deployment
I've added the 2 day buffer, for example, but the refresh button will still be using an old version of the codebase. And we ought not rush things and implement a new deployment just for that.